addons/mail/views/mail_templates.xml

```diff
@@ -282,6 +282,12 @@
             </xpath>
         </template>
 
+        <template id="qunit_mobile_suite" name="mail_mobile_tests" inherit_id="web.qunit_mobile_suite">
+            <xpath expr="//t[@t-set='head']" position="inside">
+                <script type="text/javascript" src="/mail/static/src/messaging/test_utils.js"></script>
```

Comment:
This should be moved into `mail_js_test_assets` and removed from `mail_tests` and from here.